---
title: "Parallel single-cell RNA-seq and genetic recording reveals lineage decisions in developing embryoid bodies"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- Ik Soo Kim
- Jingyi Wu
- Gilbert J. Rahme
- Sofia Battaglia
- Atray Dixit
- Elizabeth Gaskell
- admin
- Luca Pinello
- Bradley E. Bernstein

# # Author notes (optional)
# author_notes:
# - "Equal contribution"
# - "Equal contribution"

date: "2020-10-06"
doi: "10.1016/j.celrep.2020.108222"

# # Schedule page publish date (NOT publication's date).
# publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: Cell Reports
publication_short: Cell Rep.

abstract: Early developmental specification can be modeled by differentiating embryonic stem cells (ESCs) to embryoid bodies (EBs), a heterogeneous mixture of three germ layers. Here, we combine single-cell transcriptomics and genetic recording to characterize EB differentiation. We map transcriptional states along a time course and model cell fate trajectories and branchpoints as cells progress to distinct germ layers. To validate this inferential model, we propose an innovative inducible genetic recording technique that leverages recombination to generate cell-specific, timestamp barcodes in a narrow temporal window. We validate trajectory architecture and key branchpoints, including early specification of a primordial germ cell (PGC)-like lineage from preimplantation epiblast-like cells. We further identify a temporally defined role of DNA methylation in this PGC-epiblast decision. Our study provides a high-resolution lineage map for an organoid model of embryogenesis, insights into epigenetic determinants of fate specification, and a strategy for lineage mapping of rapid differentiation processes.

# # Summary. An optional shortened abstract.
# summary: An interactive pipeline capable of disentangling and visualizing complex branching trajectories from both single-cell transcriptomic and epigenomic data.

tags: [Single-cell, scRNA-seq, Embryogenesis, Lineage Tracing, Stem Cells, Trajectory Inference]

# Display this page in the Featured widget?
featured: false

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: 'https://www.sciencedirect.com/science/article/pii/S2211124720312110/pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---

<!-- {{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

{{% callout note %}}
Create your slides in Markdown - click the *Slides* button to check out the example.
{{% /callout %}}

Supplementary notes can be added here, including [code, math, and images](https://wowchemy.com/docs/writing-markdown-latex/). -->
